spec/release_spec.rb

```diff
@@ -3,7 +3,8 @@
   let(:name) { "foo" }
   let(:release) { load_release(release_data) }
   let(:version) { "3" }
-  let(:release_data) { { "name" => name, "version" => version, "git" => repo } }
+  let(:url) { "http://local.url/release" }
+  let(:release_data) { { name: name, version: version, git: repo } }
```

Comment:
release data should always come in as strings (not symbols), I would expect using symbols would brake [those assignments](https://github.com/cloudfoundry-incubator/bosh-workspace/blob/master/lib/bosh/workspace/release.rb#L6-L10)